openapi.go

```diff
@@ -263,7 +280,7 @@ func RegisterOpenAPIOperation[T, B any](openapi *OpenAPI, route Route[T, B]) (*o
 	for _, params := range route.Operation.Parameters {
 		if params.Value.In == "path" {
 			if !strings.Contains(route.Path, "{"+params.Value.Name) {
-				return nil, fmt.Errorf("path parameter '%s' is not declared in the path", params.Value.Name)
+				panic(fmt.Errorf("path parameter '%s' is not declared in the path", params.Value.Name))
```

Comment:
Why a panic?

Reply:
We don't want to declare non-existing path params. Do you think a warning/an error is enough?

Reply:
I would say if the app crash on boot, I'm OK with a panic, otherwise it could be a real problem if it occurs only on some condition like having a call that will cause a panic.

I think it's a first case, so why not

Reply:
Yeah I think a general we should is anything before start is fine too panic where it's genuinely pretty difficult to get the error to the user. Another case where I like panic is when the users configuration just should be possible. I.E providing nil as a Type etc